client/reports.lua

```diff
@@ -1,8 +1,8 @@
-local reportOptions = {
-    function(report)
+local function reportAction(selected, report)
+    if selected == 1 then
         lib.alertDialog({
             header = string.format('Report ID: %s | Sender: %s', report.id, report.senderName),
-            content = report.message,
+            content = string.format('message: %s', report.message),
```

Comment:
```suggestion
            content = ('message: %s'):format(report.message),
```
Utilize the colon synstax for formatting strings